app/form/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Home() {
+	return <main>HOME</main>;
+}
```

Comment:
There is very popular pattern in CRUD applications (and not only), where URL is descriptive to the content of the page.
So assuming that this page would be a page for creating a game, I'd suggest placing it under one of these URLs:

- `/game/create`
- `/game/add`

Once you'll have page to edit existing games, you can place it under
`/game/{game_id}/edit`